remail/controller.py

```
@@ -6,11 +6,36 @@
     User,
+    Protocol,
 )
 from datetime import datetime
 import duckdb
 import logging
 from sqlmodel import SQLModel
+from remail.email_api.service import ImapProtocol, ExchangeProtocol, ProtocolTemplate
+import remail.email_api.email_errors as errors
+import keyring
+from tzlocal import get_localzone
+from pytz import timezone
```

Ruff (0.8.2)

`pytz.timezone` imported but unused

Remove unused import: `pytz.timezone`

(F401)